### PR TITLE
[Verified members] feat: add verified members via permissions saga

### LINKED ISSUE #1729: [Verified members] Feature: implement the add verified members saga - Permissions
### Story

The add verified members form should call this saga and initiate the transaction.

### Description

The saga receives an array of members to verify and the colony to add them to.
It should update the loading, success and error states accordingly.
We can use `manageVerifiedRecipients` as a reference, but the underlying logic will probably be very different.

## Implementation

The saga receives the following payload
```JSX
{
	colonyAddress: Address;
	colonyName: string;
	members: string[];
	annotationMessage?: string;
	customActionTitle?: string;
}
```
And builds the necessary transactions (TBD when we have more info after block ingestor work is completed).
Naturally, display loading/success/error states and on success redirect to `/${colonyName}?tx=${txHash}`.

### This issue is complete when...

- [ ] The saga receives the payload
- [ ] The transaction is shown as pending until it's complete
- [ ] The transaction is shown as completed and is shown as a `CompletedAction`
- [ ] If an error happened it is displayed to the user

### PR DESCRIPTION
**Pull request by bassgeta**
## Description

This PR adds a saga to add verified members, construct the JSON object for the block ingestor, and call the saga from a very dumb component for testing purposes (will remove once we merge the form).

**New stuff** ✨

* `addVerifiedMembers` saga
* `TmpVerifiedMembers` component

Resolves  #1729

### CODE REVIEW

**Review by jakubcolony — APPROVED** [review]
LGTM, nice work! 

Just a heads up the `build` check is failing on this PR

File: src/redux/sagas/actions/addVerifiedMembers.ts [hunk 1]
```diff
@@ -1,27 +1,148 @@
...
+      addVerifiedMembers.channel,
+      ActionTypes.TRANSACTION_HASH_RECEIVED,
+    );
+
+    setTxHash?.(txHash);
+
+    yield takeFrom(
+      addVerifiedMembers.channel,
+      ActionTypes.TRANSACTION_SUCCEEDED,
+    );
```
Comment (iamsamgibbs) [inline src/redux/sagas/actions/addVerifiedMembers.ts:112]: I believe this should be replaced with `waitForTxResult` (#1018 currently in progress to replace these globally)

**Review by iamsamgibbs — CHANGES_REQUESTED** [review]
Looks good to me! Just one minor change request, have left a comment.

**Review by iamsamgibbs — APPROVED** [review]
Thanks for the update! Looks great!

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 👌

**Pull request merged**